modules/ztp-sno-du-disk-encryption.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-disk-encryption_{context}"]
+= About disk encryption with TPM and PCR protection
+
+You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. 
+
+TPM is a hardware component that stores cryptographic keys and evaluates the security state of your system. PCRs within the TPM store hash values that represent the current hardware and software configuration of your system. You can use the following PCR registers to protect the encryption keys for disk encryption:
+
+* PCR 1: Represents the Unified Extensible Firmware Interface (UEFI) state.
+* PCR 7: Represents the secure boot state. 
```

Comment:
Future suggestion: Maybe consider using a [definition list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) here?

Reply:
Updated as per the suggestion.